src/main/java/rife/database/queries/Insert.java

```diff
@@ -230,20 +220,19 @@ public Insert field(String field, boolean value) {
     public Insert field(String field, Select query) {
         if (null == query) throw new IllegalArgumentException("query can't be null.");
 
-        var buffer = new StringBuilder();
-        buffer.append("(");
-        buffer.append(query.toString());
-        buffer.append(")");
+        String buffer = "(" +
```

Comment:
Same string concatenation observation

Reply:
For SQL queries I do think this is worth preserving as a StringBuilder, this can potentially be in the critical path.